src/App.tsx

```diff
@@ -1,61 +1,152 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useEffect, useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { User } from './types/user';
+import { Todo } from './types/todo';
+import { findUserByTodoUserId } from './helpers/findUserByTodoUserId';
+
+interface FormValue {
+  userId: string;
+  title: string;
+  emptyTitleError: boolean;
+  emptyUserError: boolean;
+}
 
 export const App = () => {
+  const [users, setUsers] = useState<User[]>([]);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [formValue, setFormValue] = useState<FormValue>({
+    userId: '0',
+    title: '',
+    emptyTitleError: false,
+    emptyUserError: false,
+  });
+
+  useEffect(() => {
+    setUsers(usersFromServer);
+    setTodos(todosFromServer);
+  }, []);
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    const { title, userId } = formValue;
+    let error = false;
+
+    if (userId === '0') {
```

Comment:
preferable to use number type for id
then
```suggestion
    if (!userId) {
```